vision/classification/mnist/README.md

```diff
@@ -23,12 +25,25 @@ The model is trained in CNTK following the tutorial [CNTK 103D: Convolutional Ne
 We used CNTK as the framework to perform inference. A brief description of the inference process is provided below:
 
 ### Input
-shape `(1x1x28x28)`
+Input tensor has shape `(1x1x28x28)`, with type of float32.      
+One image a time. This model doesn't support mini-batch.      
 
 ### Preprocessing
+Images are resized into (28x28) in grayscale, with black background, white foreground(The number should be in white). Color value is scaled to [0.0, 1.0]. 
```

Comment:
nit: in grayscale, with a black background and a white foreground (the number should be in white).